.github/CHANGELOG.md

```diff
@@ -51,37 +51,52 @@
 
   A PennyLane QNode can be converted into a `torch.nn` layer using the `qml.qnn.TorchLayer` class:
 
-  ```python
-  @qml.qnode(dev)
-  def qnode(inputs, weights_0, weight_1):
-     # define the circuit
-     # ...
+  ```pycon
+  >>> @qml.qnode(dev)
+  ... def qnode(inputs, weights_0, weight_1):
+  ...    # define the circuit
+  ...    # ...
 
```

Comment:
```suggestion
  ...    # ...
```

Reply:
lol this "suggestion" is not registering as something that is committable in github